### PR TITLE
FEAT Enable/Disable all services - UI

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

- Enable/Disable All Services UI

image
image
image

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

**Review by ArmanNik — CHANGES_REQUESTED** [review]
Have you pulled from main? Modals don't have the `warning` attribute anymore, now we set the `state`

File: src/routes/console/organization-[organization]/+page.svelte [hunk 1]
```diff
@@ -38,6 +40,17 @@
         return { name, icon };
     };
 
+    function allServiceDisabled(project: Models.Projectj): boolean {
```
Comment (TorstenDittmann) [inline src/routes/console/organization-[organization]/+page.svelte:43]: typo on `Models.Projectj`?

File: src/routes/console/project-[project]/settings/+page.svelte [hunk 2]
```diff
@@ -42,6 +46,55 @@
...
+        } finally {
+            showDisableAll = false;
+            showEnableAll = false;
+        }
+    }
+
+    $: {
+        // When project name is updated, finalize the updating flow
+        $project.name;
+    }
```
Comment (TorstenDittmann) [inline src/routes/console/project-[project]/settings/+page.svelte:96]: why is this needed?

File: src/routes/console/project-[project]/settings/+page.svelte [hunk 3]
```diff
@@ -14,11 +14,15 @@
...
     import { Submit, trackEvent, trackError } from '$lib/actions/analytics';
+    import EnableAllServices from './enableAllServices.svelte';
+    import DisableAllServices from './disableAllServices.svelte';
 
     let name: string = null;
     let showDelete = false;
     const endpoint = sdk.forConsole.client.config.endpoint;
     const projectId = $page.params.project;
+    let showDisableAll = false;
+    let showEnableAll = false;
```
Comment (TorstenDittmann) [inline src/routes/console/project-[project]/settings/+page.svelte:25]: we should probably do reactive statements on those and have state manage them by itself 👍🏻

File: src/routes/console/project-[project]/settings/+page.svelte [hunk 4]
```diff
@@ -42,6 +46,55 @@
...
+        } catch (error) {
+            addNotification({
+                type: 'error',
+                message: error.message
+            });
+            trackError(error, Submit.ProjectService);
+        } finally {
+            showDisableAll = false;
+            showEnableAll = false;
+        }
```
Comment (TorstenDittmann) [inline src/routes/console/project-[project]/settings/+page.svelte:90]: if these variables are reactive statements, we can remove this logic

File: src/routes/console/project-[project]/settings/+page.svelte (hunk 3 above)
Comment (lohanidamodar) [reply src/routes/console/project-[project]/settings/+page.svelte:25]: I don't understand what you mean? This is for showing the confirmation dialog?

File: src/routes/console/organization-[organization]/+page.svelte (hunk 1 above)
Comment (lohanidamodar) [reply src/routes/console/organization-[organization]/+page.svelte:43]: fixed.

File: src/routes/console/project-[project]/settings/+page.svelte (hunk 2 above)
Comment (lohanidamodar) [reply src/routes/console/project-[project]/settings/+page.svelte:96]: removed.

File: src/routes/console/project-[project]/settings/+page.svelte (hunk 3 above)
Comment (lohanidamodar) [reply src/routes/console/project-[project]/settings/+page.svelte:25]: This is how it is implemented for Delete Project using `showDelete` ?

**Comment by lohanidamodar** [comment]
> Have you pulled from main? Modals don't have the `warning` attribute anymore, now we set the `state`

fixed

**Pull request merged**